talented-tigers/spydir/templatetags/generator_tags.py

```diff
@@ -0,0 +1,13 @@
+from django import template
```

Comment:
I would have assumed that the module generator_tags would be a generator. still, short clear, but missing docstrings makes me wonder what it would be used for.